<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recruiterProfile</class>
 <widget class="QDialog" name="recruiterProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_logOut">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="edit_jobTitle"/>
       </item>
       <item>
        <widget class="QLabel" name="label_of">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_company"/>
       </item>
      </layout>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="button_toggleEditMode">
       <property name="text">
        <string>Toggle Read</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QPlainTextEdit" name="edit_company_desc"/>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="button_reload">
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_regDate">
         <property name="text">
          <string>Registration Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_regDate"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_name"/>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_dob">
           <property name="text">
            <string>Date of Birth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_dob"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_gender">
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_gender"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="1" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_phone">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_phone"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_email">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_email">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_avatar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Avatar</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_url">
         <property name="text">
          <string>Company URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_url"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView_vacancy"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="button_addVac">
          <property name="text">
           <string>Add Vacancy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_delVac">
          <property name="text">
           <string>Delete Vacancy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableView" name="tableView_application"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="button_delApp">
          <property name="text">
           <string>Delete Applications</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
